pkg/storage/gcp/bucket.go

```diff
@@ -0,0 +1,21 @@
+package gcp
+
+import (
+	"context"
+	"io"
+)
+
+// Bucket defines the storage bucket TODO
+type Bucket interface {
+	// DeleteModule removes a module @ version from the bucket
+	DeleteModule(ctx context.Context, module, version string) error
+	// GetReader returns a reader for the module filename and an error if any
+	GetReader(ctx context.Context, filename string) (io.ReadCloser, error)
```

Comment:
`GetReader` is a bit ambiguous, what about `GetObject` or `Open` etc?

Reply:
I'me trying to decide on better names for both. They both only return the reader/writer, how about `Open` and `Write`.